gamemodes/terrortown/gamemode/shared/sh_shopeditor.lua

```diff
@@ -14,11 +14,30 @@ ShopEditor.savingKeys = {
 }
 
 ShopEditor.cvars = {
-	ttt2_random_shops = {typ = "number", bits = 8},
-	ttt2_random_team_shops = {typ = "bool"},
-	ttt2_random_shop_reroll = {typ = "bool"},
-	ttt2_random_shop_reroll_cost = {typ = "number", bits = 8},
-	ttt2_random_shop_reroll_per_buy = {typ = "bool"}
+	ttt2_random_shops = {
+		order = 1, typ = "bool", default = 0,
+		name = "random_shops", b_desc = true
+	},
+	ttt2_random_shop_items = {
+		order = 2, typ = "number", bits = 8, default = 10, min = 1, max = 60,
```

Comment:
2 ^ 8 = 256 items. Setting max to 60 doesn't makes sense tho. E.g. 2^6 would be enough for that limit as well. Same below.

Reply:
Well my idea was mainly, that the shop can show 5 x 6 = 30 items at once and is nice, having 60 was just double that.
Didnt know what the bits were used. Though that is a normal integer length, so I let it stay at 8 and set the max to something I though useful for a random shop.
So rather 6 bits or max 256?

Reply:
Well "bits" dont matter much in my implementation. So some of those params can be deleted, once the shopeditor is fully moved to the F1 Menu. Decided to set them down to 6 then.